returnn/flow.py

```diff
@@ -0,0 +1,128 @@
+from sisyphus import tk
+from sisyphus.delayed_ops import DelayedJoin
+from typing import Optional, List, Union
+
+from i6_core import rasr
+from i6_core.returnn.training import Checkpoint
+
+
+def make_precomputed_hybrid_tf_feature_flow(
+    tf_graph: tk.Path,
+    tf_checkpoint: Checkpoint,
+    feature_tensor_name: str = "data",
```

Comment:
I think I would also rename this now, for consistency, and again correctness.
Maybe `extern_data_name`?